### DISCUSSION TITLE
Removing user-name/README.md from top of profile page

### DISCUSSION THREAD

**Discussion by chris-c-thomas** [community/community#35122] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there a way to remove `chris-c-thomas/README.md` from the top of my profile page? I was messing around and trying to add these new Holopin boards but I see this above the banner and it kinda looks out of place. 

<img width="932" alt="Screenshot 2022-10-04 at 3 49 39 PM" src="https://user-images.githubusercontent.com/22353987/193925091-e19df758-b44d-4ec6-95bb-9dd95d02dbc5.png">

All that is in the README.md file is the code from Holopin. GitLab's for instance, does not display that part.

**Comment by antonblr** [comment, marked as answer]
Per [customizing-your-profile/managing-your-profile-readme](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme#prerequisites)
>GitHub will display your profile README on your profile page if all of the following are true.
>
>You've created a repository with a name that matches your GitHub username.
The repository is public.
The repository contains a file named README.md in its root.
The README.md file contains any content.

You have couple options - 
a) make https://github.com/chris-c-thomas/chris-c-thomas repo private
b) rename [https://github.com/chris-c-thomas/chris-c-thomas/README.md](https://github.com/chris-c-thomas/chris-c-thomas/blob/master/README.md)
c) make that ^ `README.md` empty

**Reply by DenverCoder1** [reply]
That won't remove the text above the banner, that would remove the entire thing.

As far as I know it isn't possible to just remove the link to the readme at the top.

**Reply by antonblr** [reply]
Oh, yes, you can't remove just the `chris-c-thomas/README.md` line. Thanks @DenverCoder1 for pointing to the question detail I missed !

**Reply by chris-c-thomas** [reply]
Thanks everyone. Kinda bummed but oh well. I copied it to GitLab and it wasn't on that one so I thought maybe they had a setting. Oh well, no biggie.

**Comment by Mehdimome** [comment]
در تاریخ چهارشنبه ۵ اکتبر ۲۰۲۲،‏ ۱:۰۴ Jonah Lawrence <
***@***.***> نوشت:
